python/src/sdf/pyCamera.cc

```diff
@@ -296,6 +299,8 @@ void defineCamera(pybind11::object module)
       .value("BAYER_BGGR8", sdf::PixelFormatType::BAYER_BGGR8)
       .value("BAYER_GBRG8", sdf::PixelFormatType::BAYER_GBRG8)
       .value("BAYER_GRBG8", sdf::PixelFormatType::BAYER_GRBG8);
+
+  GZ_UTILS_WARN_RESUME__DEPRECATED_DECLARATION
```

Comment:
do you mind adding the new APIs to [pySensor.cc](https://github.com/gazebosim/sdformat/blob/3656de304117807384a5214528a02e120e2413c6/python/src/sdf/pySensor.cc)?

Reply:
ah yes added. [d64d5f6](https://github.com/gazebosim/sdformat/pull/1454/commits/d64d5f6505697a70c57216c053c656ef1b7895ca)